### ISSUE TITLE
Google Photos completely bypass Simple Gallery fingerprint authentication

### ISSUE THREAD

**Issue by elys1an** [SimpleMobileTools/Simple-Gallery#2934]
I have fingerprint authentication set to ON when opening Simple Gallery and it works fine except on Google Photos.

Whenever I took a photo / video, right after it gets processed on the bottom right corner and I click on it, it will just open Simple Gallery straight up without fingerprint authentication, I can also swipe left and right basically viewing all the photos I have on the folder the picture / video taken was stored on.

So if the picture / video is stored on Camera, I can see all files inside that folder by swiping left and right.

OS: Android 11, App version: Latest Simple Gallery Pro.

**Comment by tibbi** [comment]
you mean Google Camera? And do you mean the "Password protect the whole application" setting or locking a specific folder?

**Comment by tibbi** [comment]
are you sure it is our gallery opening, not some other one?

**Comment by elys1an** [comment]
> are you sure it is our gallery opening, not some other one?

Yes, I have set my Google Camera to open Simple Gallery by default instead of Google Photos.

**Comment by realquink** [comment]
I get the same issue for Signal for a password protected folder with a pin code. I use the camera in Signal and can see all my password protected folders. Would be nice if there was a way for them to be hidden phone wide and not just in the simple gallery app. Thanks for reading.
